tests/phpunit/test-rest-api-post-property.php

```diff
@@ -18,6 +18,11 @@
 use WP_REST_Request;
 
 class TestRESTAPIPostProperty extends RESTAPITestCase {
+
+	public function setUp() {
+		wp_cache_flush();
+	}
+	
```

Comment:
Given you've not added/changed any tests, can this be removed.

Reply:
@addaeabeng I see we kept this, can you add an explainer here on why the feedback was dismissed without changes?